### PR TITLE
Logout functionality for stakeholder and student

### LINKED ISSUE #87: FEATURE : Logout functionality for stakeholder and student
**What the feature does:**

1. Implementing a secure logout feature for both stakeholders and students.

**Images for reference:**

**1. Student Logout Feature:** The logout option for students must be enabled to allow them to securely exit their session and protect their 
personal and academic information.
image

**2. Stakeholder Logout Feature:** The logout functionality for stakeholders must be enabled to ensure secure session termination, protecting sensitive data from unauthorized access.
image

@Harshdev098 
Pls assign me this issue

**Give me the labels of gssoc,level3,hacktoberfest,etc.**

### PR DESCRIPTION
**Pull request by ygowthamr**
### Implemented Logout Functionality for Stakeholders and Students

**Description:**
- Added logout functionality for both stakeholders and students.

**Code Changes:**
- In dBServer.js file the lines nos are **10,111**
- In Dashboard.html the line nos are **30,102-122**
- In stk_dashboard.html line nos are **223-243**
- Logout.js in entirely new file

@Harshdev098 
Give me labels like gssoc extd, level3, hacktoberfest,etc

Fixes #87

### CODE REVIEW

**Comment by ygowthamr** [comment]
@Harshdev098 
I have created a new and branch 
Pls merge my pr.

File: login-system/logout.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const logout = async (req, res) => {
+  localStorage.removeItem("accessToken");
```
Comment (Harshdev098) [inline login-system/logout.js:2]: There is no use of these endpoints or functions, you are doing same thing in the client side code

File: public/dashboard.html [hunk 2]
```diff
@@ -99,7 +99,27 @@
...
+                const token = localStorage.getItem('token'); // Retrieve the token from storage
+                const response = await fetch('/logout', {
+                    method: 'POST',
+                    headers: {
+                        'Authorization': `Bearer ${token}`, // Send the token if needed
+                    },
+                });
 
+                if (response.ok) {
+                    localStorage.removeItem('token'); // Remove the token from storage
```
Comment (Harshdev098) [inline public/dashboard.html:113]: Make them functions not any endpoints

**Review by Harshdev098 — CHANGES_REQUESTED** [review]

**Comment by ygowthamr** [comment]
@Harshdev098 
In the next issue i am going to do sessions for this app at that time i use this endpoint.

**Comment by Harshdev098** [comment]
Okay

**Pull request merged**